### PR TITLE
Add wallet commands

### PR DESCRIPTION
**Pull request by NicolasMahe**
This PR add the commands related to wallet

```
➜ ./dev-cli wallet

Manage wallets

Usage:
  mesg-core wallet [command]

Available Commands:
  create      Create a new account
  delete      Delete an account
  export      export an account
  import      import an account
  list        list accounts

Flags:
  -h, --help   help for wallet

Global Flags:
      --no-color     disable colorized output
      --no-spinner   disable spinners

Use "mesg-core wallet [command] --help" for more information about a command.
```

### CODE REVIEW

File: commands/wallet_export.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+func (c *walletExportCmd) runE(cmd *cobra.Command, args []string) error {
+	account, err := c.e.Export(c.address, c.passphrase)
+	if err != nil {
+		return err
+	}
+	b, err := json.Marshal(account)
+	if err != nil {
+		return err
+	}
+	fmt.Println(string(b))
```
[the remark this answers is not in the document]
Comment (NicolasMahe) [reply commands/wallet_export.go]: Done

File: commands/wallet_list.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+func (c *walletListCmd) runE(cmd *cobra.Command, args []string) error {
+	accounts, err := c.e.List()
+	if err != nil {
+		return err
+	}
+	if len(accounts) == 0 {
+		fmt.Println("No account")
+		return nil
+	}
+	for i, account := range accounts {
```
[the remark this answers is not in the document]
Comment (NicolasMahe) [reply commands/wallet_list.go]: Done in your feedback PR

**Comment by NicolasMahe** [comment]
@antho1404 Modification taking into account and add pretty print for the export command

File: commands/wallet_import.go [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	jsonFile   string
+	account    provider.EncryptedKeyJSONV3
+
+	e WalletExecutor
+}
+
+func newWalletImportCmd(e WalletExecutor) *walletImportCmd {
+	c := &walletImportCmd{e: e}
+	c.cmd = newCommand(&cobra.Command{
+		Use:     "import ",
```
Comment (krhubert) [inline commands/wallet_import.go]:
```suggestion
		Use:     "import",
```

**Pull request merged**